requirements/dev.in

```diff
@@ -0,0 +1,7 @@
+-c constraints.txt
+
+-r base.txt
+
+-r test.txt
```

Comment:
why adding tests and docs in dev.in ?

Reply:
We normally do that to allow running any of the tests or doc building commands directly from the development environment without needing to go through tox.  It also helps keep all the dependency versions in sync if there's one requirements file that contains essentially all of the package's dependencies.

Reply:
fro development we need all dependencies of the project.